daemons/controld/controld_transition.c

```diff
@@ -125,7 +125,8 @@ do_te_invoke(long long action,
         ha_msg_input_t *input = fsa_typed_data(fsa_dt_ha_msg);
         xmlNode *graph_data = input->xml;
         const char *ref = crm_element_value(input->msg, PCMK_XA_REFERENCE);
-        const char *graph_file = crm_element_value(input->msg, F_CRM_TGRAPH);
+        const char *graph_file = crm_element_value(input->msg,
+                                                   PCMK__XA_CRM_TGRAPH_FILE);
```

Comment:
We can just drop this (should have been done with ed845d8b82e). The XML ultimately comes from the scheduler, so it can't be from an old peer or remote node.

Reply:
Done in next push.